src/pages/404.astro

```diff
@@ -0,0 +1,108 @@
+  @@@@@@@@@@@@@               @@@@@@@@@@@@@             @@@@@@@@@@@@                    @@@@@@@@@@@@        @@@@@@@@@@@@@              @@@@@@@@@@@@@@     
+  @@@@@@@@@@@@@               @@@@@@@@@@@@@             @@@@@@@@@@@@                    @@@@@@@@@@@@        @@@@@@@@@@@@@              @@@@@@@@@@@@@@     
+  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@        @@@@@@@@@@@@                    @@@@@@@@@@@@        @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@        @@@@@@@@@@@@                    @@@@@@@@@@@@        @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@              @@@@@@@@@@@@@             @@@@@@@@@@@@        @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+                              @@@@@@@@@@@@@                    @@@@@@@@@@@@@             @@@@@@@                                        @@@@@@@@@@@@@     
+                              @@@@@@@@@@@@@                    @@@@@@@@@@@@@             @@@@@@@                                        @@@@@@@@@@@@@     
+                              @@@@@@@@@@@@@                          @@@@@@@@@@@@@@@@@@@@@                                              @@@@@@@@@@@@@     
+                              @@@@@@@@@@@@@                          @@@@@@@@@@@@@@@@@@@@@                                              @@@@@@@@@@@@@     
+    </pre>
+    <div class="type-box-container">
+      <p class="type-box" id="type-box-404"></p>
+    </div>
+  </div>
+</Layout>
+
+<script>
+  import { Typoist } from "typoist";
+  const typeIt = (selector, text, options) => {
+    const textArea = document.querySelector(selector);
+    if (textArea) {
+      const typoist = new Typoist({
+        ...options,
+        appendFunction: (character) =>
+          new Promise((resolve) => {
+            textArea.innerHTML += character;
+            resolve("appended char");
+          }),
+        deleteFunction: () =>
+          new Promise((resolve) => {
+            textArea.innerHTML = textArea.innerHTML.slice(0, -1);
+            resolve("deleted char");
+          }),
+      });
+
+      typoist.type(text);
+    }
+  };
+
+  window.onload = () => {
+    typeIt("#type-box-404", "You tried to access a forbidden page. This incident will be reported.", {
+      speed: 10,
+      mistakeProbability: 0.1,
+    });
+  };
+</script>
+
+<style>
+  pre {
+    user-select: none;
+    font-size: 1vw;
+  }
+  .ascii {
+    position: relative;
+    top: 9vh;
+    padding-left: 3.5vw;
+    opacity: 50%;
+    display: inline-block;
+  }
+  .type-box-container {
+    position: relative;
+    width: 100%;
+    padding-left: 9vw;
+    margin: 19vh 0%;
+    display: inline-block;
+  }
+  .type-box {
+    padding-top: 3.5vh;
+    font-size: 2vw;
+    font-family: "Press Start 2P", monospace;
```

Comment:
Is this font available on all systems? If not, either use a different font or load this font.